src/api-service/dev-deploy.ps1

```diff
@@ -0,0 +1,19 @@
+param (
+    [Parameter()]
+    [string] $app_dir="$PSScriptRoot",
+    [Parameter(Mandatory=$true)]
+    [string] $target
+)
+
+Set-Location "$app_dir\..\pytypes"
```

Comment:
If you use Linux paths, it will still work on Windows and the script might also work on Linux.